pkg/enactmentstatus/status.go

```diff
@@ -34,12 +34,13 @@ import (
 )
 
 var (
-	log = logf.Log.WithName("enactmentstatus")
+	log       = logf.Log.WithName("enactmentstatus")
+	allErrors = func(error) bool { return true }
 )
 
 func Update(cli client.Client, key types.NamespacedName, statusSetter func(*nmstate.NodeNetworkConfigurationEnactmentStatus)) error {
 	logger := log.WithValues("enactment", key.Name)
-	return retry.RetryOnConflict(retry.DefaultRetry, func() error {
+	return retry.OnError(retry.DefaultRetry, allErrors, func() error {
```

Comment:
I would add a comment explaining how temporary network unavailability needs to be accounted for when handling errors. Is this the only place where this problem can happen? Or is more generic handling needed? Isn't connectivity verified before this update is attempted? Or is this when we try to update after rollback?

Reply:
Hey @jcaamano, AFAIR you were working on some bug when this was discovered. Do you have a Jira handle? We would simply link it in the commit so that whoever is interested at any time would `git blame` and read what's needed

Reply:
This was something @qinqon tried out in context of https://issues.redhat.com//browse/OPNET-268

Reply:
@jcaamano @mkowalski added some comments to explain it.